requirements.txt

```diff
@@ -4,3 +4,4 @@ numpy===1.24.4; python_version <= "3.8"
 numpy==1.26.4; python_version > "3.8"
 pgvector==0.3.2
 SQLAlchemy[asyncio]==2.0.34
+Pillow==10.4.0
```

Comment:
Where is this required? We may have to add this as a dependency to the library in pyproject.toml

Reply:
It is required to the testing files only and I have removed it from here to specify in pyproject.toml's test dependencies.